sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -67,12 +67,15 @@ case class CreateDatabaseCommand(
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
     val catalog = sparkSession.sessionState.catalog
+    // if the path is specified by the user then need to consider DB as external DB
```

Comment:
nit: If the location is specified by the user then need to consider DB as external DB,